tests/Unit/Ads/AccountServiceTest.php

```diff
@@ -461,16 +461,18 @@ public function test_get_billing_status_pending() {
 	}
 
 	public function test_disconnect() {
-		$this->options->expects( $this->exactly( 7 ) )
+		$this->options->expects( $this->exactly( 9 ) )
 			->method( 'delete' )
 			->withConsecutive(
 				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ],
 				[ OptionsInterface::ADS_ACCOUNT_STATE ],
 				[ OptionsInterface::ADS_BILLING_URL ],
 				[ OptionsInterface::ADS_CONVERSION_ACTION ],
+				[ OptionsInterface::ADS_CUSTOMER_DATA_TERMS ],
+				[ OptionsInterface::ADS_ENHANCED_CONVERSION_STATUS ],
 				[ OptionsInterface::ADS_ID ],
 				[ OptionsInterface::ADS_SETUP_COMPLETED_AT ],
-				[ OptionsInterface::CAMPAIGN_CONVERT_STATUS ]
+				[ OptionsInterface::CAMPAIGN_CONVERT_STATUS ],
```

Comment:
Did this pass in the unit tests? A trailing comma in a function is PHP 8.0+

Edit: Nevermind, function calls allow it since PHP 7.3 it's just the function declaration that's 8.0+